ui/src/shared/components/Dygraph.tsx

```diff
@@ -46,11 +46,10 @@ import {
   DygraphData,
   DygraphClass,
   DygraphSeries,
-  Constructable,
 } from 'src/types'
 import {LineColor} from 'src/types/colors'
 
-const Dygraphs = D as Constructable<DygraphClass>
+const Dygraphs = D as any
```

Comment:
is there a more specific type than any?

Reply:
We wrap our imports of dygraphs in a few levels to facilitate some dygraphs plugins. TypeScript seems to have trouble inferring that `D` is constructible because of it. I tried a few different things here but this seemed like the most reasonable—the `any` annotation only affects one line across the rest of the codebase.